### PR TITLE
Ballot results for XML Web Services and Transactions

### PR DESCRIPTION
**Pull request by dblevins**

### CODE REVIEW

File: transactions/2.0/_index.md [hunk 1]
```diff
@@ -32,19 +32,21 @@ Please reference that ballot for the official results.
...
+| Scott Stark, Mark Little                            | Red Hat             |  +1  |
+| David Blevins, Jean-Louis Monteiro                  | Tomitribe           |  +1  |
+| Ivar Grimstad                                       | EE4J PMC            |  +1  |
+| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |  +1  |
+| Werner Keil                                         | Committer Members   |  +1  |
+| Scott (Congquan) Wang                               | Enterprise Members  |  +1  |
+|                                                     | Total               |  10  |
+
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline transactions/2.0/_index.md:52]: Need a link to the original ballot request from the mailing list archive.

File: xml-web-services/3.0/_index.md [hunk 2]
```diff
@@ -35,19 +35,21 @@ Please reference that ballot for the official results.
...
+| Scott Stark, Mark Little                            | Red Hat             |  +1  |
+| David Blevins, Jean-Louis Monteiro                  | Tomitribe           |  +1  |
+| Ivar Grimstad                                       | EE4J PMC            |  +1  |
+| Marcelo Ancelmo, Martijn Verburg                    | Participant Members |  +1  |
+| Werner Keil                                         | Committer Members   |  +1  |
+| Scott (Congquan) Wang                               | Enterprise Members  |  +1  |
+|                                                     | Total               |  10  |
+
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline xml-web-services/3.0/_index.md:55]: Need a link to the original ballot request from the mailing list archive.

**Review by kwsutter — COMMENTED** [review]
Missed the ballot urls.  Otherwise, ready to go.

**Comment by kwsutter** [comment]
Bump, @dblevins...  Can we get these items cleared up so that we can merge these today?  thanks!

File: transactions/2.0/_index.md (hunk 1 above)
Comment (cesarhernandezgt) [reply transactions/2.0/_index.md]: I give it a try at: 9cebe76

File: xml-web-services/3.0/_index.md (hunk 2 above)
Comment (cesarhernandezgt) [reply xml-web-services/3.0/_index.md]: I give it a try at: 9cebe76

Comment (ivargrimstad) [reply xml-web-services/3.0/_index.md]: LGTM

**Pull request merged**